Kitodo-LongTimePreservationValidationModule/pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>kitodo-production</artifactId>
+        <groupId>org.kitodo</groupId>
+        <version>3.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <name>Kitodo − Long Time Preservation Validation Module</name>
```

Comment:
Please use a normal dash `-` as separator and not the long dash `−`. This is already happend in your Image Management Module.

Reply:
The usage of the "en dash" is [correct here. (Link in German)](https://typefacts.com/artikel/binde-und-gedankenstrich#8)

Reply:
I will change it, but if there is an issue with it (unicode problems?) we should fix them.

Citing form [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s2.3.3-non-ascii-characters):
> **Tip:** Never make your code less readable simply out of fear that some programs might not handle non-ASCII characters properly. If that should happen, those programs are **broken** and they must be **fixed**.

Reply:
That's how looks current build - is there something not matching in this log?

[INFO]
[INFO] Kitodo Production .................................. SUCCESS [  0.278 s]
[INFO] OpacPica-Plugin .................................... SUCCESS [  2.863 s]
[INFO] Kitodo - API ....................................... SUCCESS [  2.438 s]
[INFO] Kitodo - Data Management ........................... SUCCESS [  4.114 s]
[INFO] Kitodo - Data Access ............................... SUCCESS [  8.282 s]
[INFO] Kitodo - Data Format ............................... SUCCESS [  3.208 s]
[INFO] Kitodo - Data Editor ............................... SUCCESS [  4.912 s]
[INFO] Kitodo ? Image Management Module ................... SUCCESS [  0.414 s]
[INFO] Kitodo - Persistent Identifier ..................... SUCCESS [  0.507 s]
[INFO] Kitodo - Docket .................................... SUCCESS [  2.520 s]
[INFO] Kitodo - Command ................................... SUCCESS [  0.976 s]
[INFO] Kitodo - File Management ........................... SUCCESS [  1.188 s]
[INFO] Kitodo - Schema Converter .......................... SUCCESS [  0.165 s]
[INFO] Kitodo - Validation ................................ SUCCESS [  0.163 s]
[INFO] Kitodo - Core ...................................... SUCCESS [07:21 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS

Reply:
I have changed Image Management Module as well.
It seems there is missing UTF-8 support somewhere